test/zipWith.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+
+test('zipWith', function() {
+
+  eq(typeof S.zipWith, 'function');
+  eq(S.zipWith.length, 3);
+  eq(S.zipWith.toString(), 'zipWith :: (a -> b -> c) -> Array a -> Array b -> Array c');
+
+  eq(S.zipWith(S.add, [1, 2, 3], [3, 2, 1]), [4, 4, 4]);
+  eq(S.zipWith(S.add, [1, 2, 3, 4], [3, 2, 1]), [4, 4, 4]);
+  eq(S.zipWith(S.add, [1, 2, 3], [3, 2, 1, 0]), [4, 4, 4]);
```

Comment:
The problem with using `S.add` is that addition is commutative. Using a noncommutative combining function would make the tests more valuable, as it would ensure that the function is applied to its arguments in the right order. In fact, it would be ideal to use a combining function which takes arguments of different types.

Reply:
Now I an example with S.gte! But don’t fear to tell me if you’d rather have examples with other specific functions 🤔 my point of reference is definitely not helping my creativity.